.github/workflows/ci.yml

```diff
@@ -225,22 +260,72 @@ jobs:
         run: |
           git config --unset-all '${{ steps.setup_git_auth.outputs.config_key }}'
           git submodule foreach git config --unset-all '${{ steps.setup_git_auth.outputs.config_key }}'
+                    
+          echo ${{ secrets.PAT_REVIEW }} | gh auth login --with-token
```

Comment:
This doesn't look like it belongs in a clean-up step

Reply:
there is no other place to put it unless I duplicate the code and I don't want to put another step.